src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -271,8 +271,16 @@ public Locomotive click(By by) {
     }
 
     public Locomotive click(WebElement element) {
-        element.click();
-        return this;
+        try {
+            element.click();
+            return this;
+        } catch (Exception e) {
+            if (e instanceof NoSuchElementException) {
+                throw new NoSuchElementException("Unable to locate element: " + element.toString());
+            } else {
+                throw e;
+            }
+        }
```

Comment:
Per our discussion @aaron-goff, you can eliminate the `instanceof` check and the `if/else` block by using multiple `catch` blocks.

For example, here you could do something like this...
`
    ...
        } catch (NoSuchElementException nse) {
            throw new NoSuchElementException("Unable to locate element: " + element.toString());
        } catch (Exception ex) {
            throw ex;
        }
`

Reply:
Addressed in 569b124